backend/app/evaluation/toxicity/run.py

```
@@ -0,0 +1,98 @@
+        df = df.drop(columns=[f"{validator_name}_result"])
+
+    pred_cols = ["y_true"] + [f"{v}_pred" for v in VALIDATORS]
+    write_csv(
+        df[[text_col, *pred_cols]],
+        OUT_DIR / f"predictions_{dataset_name}.csv",
+    )
+    write_json(all_metrics, OUT_DIR / f"metrics_{dataset_name}.json")
+
+
+for dataset_name, dataset_cfg in DATASETS.items():
+    print(f"Evaluating dataset: {dataset_name}")
```

Ruff (0.15.9)

[warning] `print` found

Remove `print`

(T201)